Project-1/JCheung-P1-README.md

```diff
@@ -0,0 +1,474 @@
+| F | F | T | T |             T             |
+| F | T | F | F |             F             |
+| F | T | F | T |             F             |
+| F | T | T | F |             T             |
+| F | T | T | T |             T             |
+| T | F | F | F |             F             |
+| T | F | F | T |             T             |
+| T | F | T | F |             F             |
+| T | F | T | T |             T             |
+| T | T | F | F |             F             |
+| T | T | F | T |             T             |
+| T | T | T | F |             T             |
+| T | T | T | T |             T             |
+
+
+	
+**2. A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:**
+
+Conjunction normal form is a conjunction (^). We will use this to connect disjunctions. 
+  
+(a) (A→B)→C = (~A ∨ B) → C = ~(~A ∨ B) ∨ C = (~~A ∧ ~B) ∨ C = (A ∧ ~B) ∨ C = **(A ∨ C) ∧ (~B ∨ C)** [Distribution rule]  
+    
+(b) (A→(B∨C))∨(C→¬A) = **A ∨ ~A**
+
+According to this truth table, this sentence is a tautology. Thus, this sentence can be reduced to A ∨ ~A.
+
+
+| A | B | C | ((¬A → B) ∨ ((A ∧ ¬C) → B)) |   |
+|---|---|---|-----------------------------|---|
+| F | F | F | T                           |   |
+| F | F | T | T                           |   |
+| F | T | F | T                           |   |
+| F | T | T | T                           |   |
+| T | F | F | T                           |   |
+| T | F | T | T                           |   |
+| T | T | F | T                           |   |
+| T | T | T | T                           |   |
+
+
+(c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A = **A ∨ ~A**
+
+According to this truth table, this sentence is a tautology. Thus, this sentence can be reduced to A ∨ ~A.
+
+| A | B | C | ((¬A ∧ (¬B ∧ C)) ∨ ((¬A ∧ ¬C) ∨ ((B ∧ C) ∨ A))) |
+|:-:|:-:|:-:|:-----------------------------------------------:|
+| F | F | F |                        T                        |
+| F | F | T |                        T                        |
+| F | T | F |                        T                        |
+| F | T | T |                        T                        |
+| T | F | F |                        T                        |
+| T | F | T |                        T                        |
+| T | T | F |                        T                        |
+| T | T | T |                        T                        |
+
+A tautology takes the form of P ∨ ~P. 
+  
+**3. Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why:**  
+
+(a)  B(x,y) that says that x is a brother of y:
+  
+**B(xy) = ∃z(P(z,x) ∧ P(z,y)) ∧ ~F(x)**
```

Comment:
Let x=y, ~F(x), and P(z,x). Then B(x,x) follows, i.e. then someone is their own brother.